OLS-403: tekton pipeline and code (#2)
openshift/lightspeed-rag-content: tekton/README.md

# LightSpeed RAG tetkon pipeline  
This repo contains Tekton pipeline for creation of  LLM embedding using Llama_index 
 
-  Loading data
-  Indexing 
-  Storing
-  Evaluating

## Prerequisites  
 - Tekton 
 - OpenShift 
 - Kustomize CLI
  
## Usage

- Add secrets resources to the secret folder and to link the secretes to the pipeline service account 
- Copy and edit the pipeline configuration file `pipeline/data-pipeline.yaml` 

> - add you file adn resources to th `kustomization.yaml` file 

Reviewer: ```suggestion
- add you file and resources to the `kustomization.yaml` file 
```
